scheds/rust/scx_lavd/src/bpf/main.bpf.c

```diff
@@ -673,6 +678,27 @@ static void update_sys_cpu_load(void)
 		load_actual += cpuc->load_actual;
 		load_ideal += cpuc->load_ideal;
 
+		/*
+		 * Accumulate task's latency criticlity information.
+		 *
+		 * Updating cpuc->* is racy. However, it could degrade the
+		 * accuracy a little bit in very rare cases, so we do embrace
+		 * embrace for performance on purpose.
```

Comment:
Double `embrace`. I'm having a bit of difficulty understanding what the comment is trying to say. Is it saying that while updating cpu->* is racy, the resulting impact on accuracy should be small and very rare and thus should be fine?

Reply:
Yes. Thanks for the suggestion. I will revise the sentence.